lib/fastlane/plugin/wpmreleasetoolkit/version.rb

```diff
@@ -1,5 +1,5 @@
 module Fastlane
   module Wpmreleasetoolkit
-    VERSION = '0.17.1'
+    VERSION = '0.17.2'
```

Comment:
I would suggest that we bump to `0.18.0` instead, because this new version not only contains bug fixes (patch bump) but also new features (the new action), which according to semver should lead to minor bump.

Reply:
I agree here and will bump this to 0.18 instead